Carbon.MassTransit/IServiceCollectionConfiguratorExtensions.cs

```diff
@@ -212,13 +212,15 @@ public static void AddAsyncRequestResponsePatternForRequestor<T>(this IServiceCo
                 {
                     busFactoryConfig.ReceiveEndpoint("request-starter-state", e =>
                     {
+                        e.AddAsHighAvailableQueue(configuration);
```

Comment:
Why it's enabled by default on there? Shouldn't we also enable quorums on usual "AddRabbitMqBus" methods?